### DISCUSSION TITLE
Newb Q - Possible to build a framework for macOS  rather than an app?

### DISCUSSION THREAD

**Discussion by vade** [pyinstaller/pyinstaller#8389] [Help]
Hello.

I have a native macOS app I am developing, and I'd like to be able to embed a python runtime and dependencies for a script, so I can run it from within my applications executable bundle without requiring end users to install anything or fuss with their main install.

Does PyInstaller support this use case?

Thanks

**Comment by rokm** [comment]
> Does PyInstaller support this use case?

No, you can only build executables.

**Reply by vade** [reply]
Thanks! That saves me time!
